.github/workflows/system-tests.yml

```diff
@@ -372,7 +372,7 @@ jobs:
             if: always() && steps.put_report.outcome == 'success'
             env:
               REPORT_NEOFS_URL: ${{ steps.put_report.outputs.REPORT_NEOFS_URL }}index.html
-            uses: Sibz/github-status-action@v1
+            uses: guibranco/github-status-action-v2@v1.1.13
```

Comment:
That's different. Are we sure it's OK to use?

Reply:
the old one is deprecated

Reply:
Looks like not a lot of choices at the moment.